cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -133,10 +133,9 @@ func runExportWorkflow(ctx context.Context, exportWorkflowPath string, varMap ma
 }
 
 // Run runs export workflow.
-func Run(clientID string, destinationURI string, sourceImage string, format string,
-	project string, network string, subnet string, zone string, timeout string,
-	scratchBucketGcsPath string, oauth string, ce string, gcsLogsDisabled bool,
-	cloudLogsDisabled bool, stdoutLogsDisabled bool, labels string, currentExecutablePath string) error {
+func Run(clientID, destinationURI, sourceImage, format, project, network, subnet, zone, timeout,
```

Comment:
I find the removal of types except for the last variable in a sequence less readable as I have to read the list until the last variable in the sequence. It can also be potentially more risky when adding new variables as type is implied by the last variable.

Reply:
Good point. Let me rollback this change.